src/main/java/org/traccar/protocol/XexunProtocolDecoder.java

```diff
@@ -79,6 +79,9 @@ private String decodeStatus(Position position, String value) {
                 case "help me!":
                     position.set(Position.KEY_ALARM, Position.ALARM_SOS);
                     break;
+                case "help me":
+                    position.set(Position.KEY_ALARM, Position.ALARM_SOS);
```

Comment:
You should not duplicate code here. Add second switch case for exiting SOS alarm option.

Reply:
You mean a new switch case with only one option? Why?
```            switch (value.toLowerCase()) {
                case "help me":
                    position.set(Position.KEY_ALARM, Position.ALARM_SOS);
                    break;
                default:
                    break;
            }```

Reply:
This is what I meant:
```java
case "help me!":
case "help me":
    position.set(Position.KEY_ALARM, Position.ALARM_SOS);
    break;
```

Reply:
sorry! I understand.
Thanks for your patience!